casino/usecase/usecase.go

```diff
@@ -23,3 +23,16 @@ func (a *CasinoUseCase) GetCasino(ctx context.Context, casinoId uint64) (*models
 func (a *CasinoUseCase) AllCasinos(ctx context.Context) ([]*models.Casino, error) {
 	return a.casinoRepo.AllCasinos(ctx)
 }
+
+func (a *CasinoUseCase) GetCasinoGames(ctx context.Context, casinoName string) ([]*models.CasinoGame, error) {
```

Comment:
Maybe when usecase is a simple proxy, we should not use it? And use repositories directly